examples/04_tool_inference_lock/ex_tool_inference_lock/main.rb

```diff
@@ -0,0 +1,199 @@
+# Copyright 2021 Trimble Inc
+# Licensed under the MIT license
+
+require 'sketchup.rb'
+require_relative 'vendor/inference_lock.rb'
+
+module Examples
+  module ToolInferenceLock
+
+    class LineTool
+      # Embed the InferenceLock module into this tool class.
+      # See vendor/inference_lock.rb for what it expects of the tool.
+      include InferenceLock
+
+      # Threshold in logical screen pixels for when the mouse is considered to
+      # be dragged.
+      APERTURE = 10
```

Comment:
Same comment here about the name; `DRAG_THRESHOLD` would be more descriptive and less ambigous.